### DISCUSSION TITLE
Add a COMING SOON video

### DISCUSSION THREAD

**Discussion by fsan2val** [owncast/owncast#3731] [Q&A]
![Pro-TV](https://github.com/owncast/owncast/assets/69484466/99259509-8445-4273-a157-fe613b9c6e62)

I don't know if what I'm going to propose is possible now:

Add a short video or an image while waiting for the connection, currently only the short text description appears, which for me is very poor, you should be able to insert a video that covers the entire screen, something similar when the Owncast logo appears when we are offline.

This would be very good since sometimes we have transmission in distant places and while the next connection is listed or adjustments are made we cannot see which transmission is coming now.

I don't know if I explained myself

Finally, add a video while waiting for the connection or place an image that covers the entire video screen

Greetings

**Comment by gabek** [comment]
The offline banner accepts markdown and html, so you can add your own rich text or multimedia if you wish.

**Reply by fsan2val** [reply]
thanks for your answer.

I know that you can put HTML but the image is small, or it moves to the left if you use a larger one, at least I have not been able to make it fit or cover the entire width of the video screen.

If you know I would appreciate a code that allows you to cover the image where the video appears.

**Reply by gabek** [reply]
I can't give you something specific to your use case, but Owncast supports custom CSS (and Javascript), so in theory, if you want to change any of the layout on your page you can do so within the admin.
